content/efcore/getting-started/querying.md

```diff
@@ -0,0 +1,47 @@
+# Querying Data
+
+Sometimes you really need an item (or items) from your database based off a certain set of parameters. In case you've forgotten, this is what querying is.
+
+Querying with EF Core uses LINQ statements (if you need some more pointers, check [this](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) out). For those who really like LINQ, [101 LINQ Samples](https://code.msdn.microsoft.com/101-LINQ-Samples-3fb9811b) is a great resource.
+
+The basic format of queries is:
+
+```c#
+variableToStoreQueryResult = context.Model.QueryKeyWord(m => m.Identifier == value)
+```
+
+## Loading All Data
+
+Grabs the entirety of a table.
+
+```c#
```

Comment:
At this time, we weren't sure how we were going to do the code portions yet, so I left it as static content. I was going to add the `{.REPL}` and `{.snippet}` blocks after we had it figured out, so I can add them in now.